mlops/bicep/modules/resources/storage_private.bicep

```diff
@@ -0,0 +1,170 @@
+// Creates a storage account, private endpoints and DNS zones
+@description('Azure region of the deployment')
+param location string
+
+@description('Tags to add to the resources')
+param tags object
+
+@description('Name of the storage account')
+param storageName string
+
+@description('Resource ID of the subnets allowed into this storage')
+param subnetIds array
+
+@allowed(['Enabled','vNetOnly','Disabled'])
+@description('Allow or disallow public network access to Storage Account.')
+param publicNetworkAccess string = 'Disabled' // for Disabled, you'd need to create private endpoints
+
+@allowed([
```

Comment:
Fairly minor, but should we include the SKU's with replication into another region?

Reply:
Not sure if this maps with an FL scenario, let's add it later if someone is asking for it?